blog/_posts/2021-12-20-discord.md

```diff
@@ -0,0 +1,176 @@
+---
+layout: blog-detail
+post-type: blog
+by: Seth Tisue
+title: "Scala Chat: Hello Discord (So Long, Gitter)"
+---
+
+On behalf of the Scala organization, I am pleased to announce that we
+have adopted **Discord** as our official chat platform.
+
+## Join us!
+
+**Please join us** for live text-based chat on the Scala Discord
+server, via this link:
+
+* https://discord.com/invite/scala
+
+The server has existed for several years now and is already
+busy and well-established.  It is actively moderated under the
+[Scala Code of Conduct](https://www.scala-lang.org/conduct/).
+
+Popular channels on the server include \#scala-users,
+\#scala-contributors, \#announcements, \#jobs, \#tooling, \#scala-js,
+\#scala-native, and others.  New channels can be proposed on \#admin.
+
+## What about Gitter?
+
+Adopting Discord means we have moved away from Gitter.  Most of our
+old Gitter rooms are now closed.  The main room remains open, but
+mainly so we can help people find us on Discord.
+
+Gitter served the Scala community extremely well for more than six
+years.  We applaud the Gitter team for their pioneering efforts, which
+revolutionized public online chat for programmers.  We also thank
+Gitter for providing us with free, high-quality service for so long.
+
+## How is Discord different?
+
+A Discord "server" may contain many "channels", also called rooms.
+
+When you join a Discord server, all of the public channels on that
+server are immediately visible to you. Unlike Gitter, you don't have
+to join each channel individually.  (But you can individually mute
+channels that don't interest you.)
+
+There is a desktop client and a mobile client (for iOS and Android).
+Both are high quality.
+
+Former Gitter users will notice a number of quality-of-life
+and user-interface improvements such as reactions and replies.
+
+## What other chat servers exist?
+
+Scala enthusiasts might also be interested in joining the following
+Discord servers:
+
+* **[Scalameta](https://discord.gg/RFpSVth)**: Scalameta-based tooling: Metals, Scalameta, Scalafix, Scalafmt, and Mdoc
```

Comment:
I only ask because I've made this mistake before myself. These are all "Never expire" links right, not the default ones that expire after 7 days?

Reply:
All the invite URLs come from the projects' official websites and are clickable at least as far as seeing the invite, so I think it's okay.